frontend/src/pages/stakingOverview/Chains.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Chain } from './Chain'
+
+export const Chains = (props) => {
+
+  return (
+    <>
+      {
+        props.chains.map((chain) => <Chain chain={chain} key={chain.chainName} />)
```

Comment:
ditto. Add key to each list element

Reply:
This is a bit confusing...
I think I already added key for the list item
<Chain chain={chain} key={chain.chainName}
(the Chain component is the list Item, though will be removing the separate component as suggested)